kustomize/kustomize_generator.go

```diff
@@ -591,3 +614,21 @@ func copyFile(src, dst string) (err error) {
 
 	return
 }
+
+func isLocalRelativePath(path string) bool {
```

Comment:
It would be nice to have a test for this function.

Reply:
Added. the function is now public because it can't be accessed from the test package otherwise.